### PR TITLE
[es] kubernetes - spanish translation

### PR DESCRIPTION
**Pull request by krol3**
Related the issue: https://github.com/open-telemetry/opentelemetry.io/issues/4887

### CODE REVIEW

**Comment by krol3** [comment]
@ramrodo @theletterf @electrocucaracha, could you review please?

**Review by ramrodo — COMMENTED** [review]
LGTM :)

File: content/es/docs/kubernetes/_index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: OpenTelemetry con Kubernetes
+linkTitle: Kubernetes
+weight: 350
+description: Usando OpenTelemetry con Kubernetes
+---
+
+## Introducción
+
+[Kubernetes](https://kubernetes.io/) es un sistema de código abierto para automatizar
+la implementación, el escalamiento y la gestión de aplicaciones en contenedores. Se ha convertido
```
Comment (theletterf) [inline content/es/docs/kubernetes/_index.md:12]:
```suggestion
la implementación, el escalado y la gestión de aplicaciones en contenedores. Se ha convertido
```

"Escalado" seems to be more prevalent in official docs (Google, AWS).

**Review by theletterf — APPROVED** [review]

**Pull request merged**